superset-frontend/package-lock.json

```diff
@@ -45172,16 +45160,18 @@
       }
     },
     "node_modules/react-map-gl": {
-      "version": "4.1.16",
-      "resolved": "https://registry.npmjs.org/react-map-gl/-/react-map-gl-4.1.16.tgz",
-      "integrity": "sha512-EtiHCeqM69wKR9RDyLvtk6pTPS5+OFeAPIsYw6afnlGTauFAq3iD40SHuAOElgoJmm7J+cjPfHqu7m7tB4/FfA==",
+      "version": "6.1.19",
+      "resolved": "https://registry.npmjs.org/react-map-gl/-/react-map-gl-6.1.19.tgz",
+      "integrity": "sha512-rrDoRyEIGzVLUB5QfgsZ5xCw7jeUtmmYzHUv86xDx8oGp90VTV2KTQJ4RPQiSAmpfIFh6/pPqI28Pguf1u/mOg==",
       "dependencies": {
         "@babel/runtime": "^7.0.0",
-        "mapbox-gl": "~0.54.0",
-        "mjolnir.js": "^2.2.0",
+        "@types/geojson": "^7946.0.7",
+        "@types/mapbox-gl": "^2.0.3",
+        "mapbox-gl": "^2.3.0",
+        "mjolnir.js": "^2.5.0",
         "prop-types": "^15.7.2",
-        "react-virtualized-auto-sizer": "^1.0.2",
```

Comment:
If you see `react-virtualized-auto-sizer` is removed in `react-map-gl`
Can you remove `"react-virtualized-auto-sizer": "^1.0.2",` in package.json? Because this lib no longer used anywhere